src/meta/src/manager/relation.rs

```diff
@@ -14,22 +14,24 @@
 
 use std::collections::HashMap;
 
-use risingwave_pb::catalog::{Index, Sink, Table};
+use risingwave_pb::catalog::{Index, Sink, Source, Table};
 
 // This enum is used in order to re-use code in `DdlServiceImpl` for creating MaterializedView and
 // Sink.
 #[derive(Debug)]
-pub enum Relation {
+pub enum StreamingJob {
     Table(Table),
```

Comment:
The term `Table` becomes more confusing to me after we add a `MaterializedSource` varaint. Strongly suggest something like `MaterializedView`. 🥵